tools/docker/Dockerfile_formatting

```diff
@@ -8,7 +8,6 @@ RUN bash buildifier.sh
 COPY tools/ci_build/install/clang-format.sh ./clang-format.sh
 RUN bash clang-format.sh
 
-RUN mkdir /addons
```

Comment:
I didn't know doing WORKDIR created a directory! :) Good to know, it'll save us some typing in the future.